plugin/xmark.vim

```diff
@@ -37,8 +37,8 @@ let s:files = {
 \ 'access': s:dir . '/applescript/accessibility.scpt',
 \ 'xsize':  s:dir . '/ext/xsize'
 \ }
-let s:app = 'Google Chrome'
 let s:tmp = {}
+let s:app = 'Google Chrome'
```

Comment:
I'm sorry for being picky, but could you exclude this unnecessary change from the commit?